sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/LineLength.vb

```diff
@@ -6,8 +6,12 @@ Namespace Tests.Diagnostics
     Public Class LineLength
         Public Sub LineLength()
 
-            Console.WriteLine("This is OK...") ' With these comments...........................................................
-            Console.WriteLine() ' Noncompliant   {{Split this 128 characters long line (which is greater than 127 authorized).}}

```

Comment:
FYI if there is multiple spaces between `Noncompliant` and `{{` for vbnet then the expected issue is ignored. I have reviewed other test cases and this doesn't appear anywhere else.

Reply:
Please fix that or create a ticket. It is a very easy mistake to accidentally add extra space.

Reply:
I readded the tests in the other PR, a fix should be straightforward